packages/chain-adapters/src/types.ts

```diff
@@ -83,43 +83,16 @@ export type SubscribeTxsInput = {
   accountType?: UtxoAccountType
 }
 
-export type TxFee = {
-  assetId: string
-  value: string
-}
+export type TransferType = unchained.TransferType
+export type TradeType = unchained.TradeType
```

Comment:
We'll probably want to also expose the TxMetadata types to make consumption easier https://github.com/shapeshift/web/pull/2257/files#diff-1a8c76a31242bf7e97ec874047f2c1dcfb6d4e47d9d1c4811b7cc8db277f9bbaR3-R4

Reply:
Yeah if necessary. I would keep them localized to their appropriate chain scope for the chain specific versions (evm.TxMetadata, cosmos.TxMetadata). I also wonder if this would be necessary if using the discrimination string, but if creating type guards, makes sense